packages/web-components/src/components/va-alert-sign-in/va-alert-sign-in.tsx

```diff
@@ -0,0 +1,310 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Prop,
+  h,
+} from '@stencil/core';
+import classnames from 'classnames';
+import { VariantNames } from './VariantNames';
+
+/**
+ * @componentName Alert - Sign In
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+@Component({
+  tag: 'va-alert-sign-in',
+  styleUrl: 'va-alert-sign-in.scss',
+  shadow: true,
+})
+export class VaAlertSignIn {
+  @Element() el!: any;
+
+  /**
+   * Determines the text content and border/background color.
+   */
+  @Prop() variant: VariantNames = VariantNames.signInRequired;
+
+  /**
+   * If `true`, doesn't fire the CustomEvent which can be used for analytics tracking.
+   */
+  @Prop() disableAnalytics?: boolean = false;
+
+  /**
+   * If `true`, the alert will be visible.
+   */
+  @Prop() visible?: boolean = true;
+
+  /**
+   * For the 'optional' variant, how long the respondent has to submit their form
+   */
+  @Prop() timeLimit?: string = '15 minutes';
+
+  /**
+   * For the 'optional' variant, the link to the form to complete without signing in
+   */
+  @Prop() noSignInLink?: string;
+
+  /**
+   * Fires when the component has successfully finished rendering for the first
+   * time.
+   */
+  @Event({
+    eventName: 'va-component-did-load',
+    composed: true,
+    bubbles: true,
+  })
+  vaComponentDidLoad: EventEmitter;
+
+  /**
+   * The event used to track usage of the component. This is emitted when an
+   * anchor link is clicked and disableAnalytics is not true.
+   */
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  private handleAlertBodyClick(e: MouseEvent): void {
+    if (!this.disableAnalytics) {
+      const target = e.target as HTMLElement;
+
+      // If it's a link being clicked, dispatch an analytics event
+      if (target?.tagName === 'VA-LINK') {
+        const innerText = target.shadowRoot.querySelector('a').innerText;
+        const detail = {
+          componentName: 'va-alert-sign-in',
+          action: 'linkClick',
+          details: {
+            clickLabel: innerText,
+            variant: this.variant,
+          },
+        };
+        this.componentLibraryAnalytics.emit(detail);
```

Comment:
Analytics will need to be coordinated with that platform team so we might just want to remove that logic for now and create a follow-up ticket to work with them.

Reply:
I can comment out the analytics logic I think.

Reply:
I think I would prefer just removing code instead of commenting it out. There are plenty of other analytics examples we can reference in other components if we want to re-add it.